config/query/boosting.yml

```diff
@@ -1,10 +1,10 @@
 base:
   format:
-    # Mainstream formats
+    # Govuk formats
```

Comment:
I think both of these comments are misleading, we should just get rid of them

Reply:
removed